sway/tree/view.c

```diff
@@ -832,6 +832,25 @@ static size_t append_prop(char *buffer, const char *value) {
 	return strlen(value);
 }
 
+static char *escape_title(char *buffer) {
+	size_t length = escape_markup_text(buffer, NULL);
+	char *escaped_title = calloc(length + 1, sizeof(char));
+	escape_markup_text(buffer, escaped_title);
+	free(buffer);
+	return escaped_title;
+}
+
+static const char *escape_if_pango(const char *buffer) {
+	// if using pango_markup in font, we need to escape all markup char
+	// from values to avoid messing with pango markup
+	if (!config->pango_markup) {
+		return escape_title(strdup(buffer));
```

Comment:
This leaks the value returned by `escape_title`. Maybe we should move this to `append_prop`?

Reply:
fixed as suggested